src/main/java/appeng/tile/storage/TileIOPort.java

```diff
@@ -136,7 +126,8 @@ public void writeToNBT_TileIOPort( final NBTTagCompound data )
 	public void readFromNBT_TileIOPort( final NBTTagCompound data )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00100)